lib/server.js

```diff
@@ -216,6 +216,15 @@ function handleRequest(opts, req, resp) {
         headers: req.headers
     };
 
+    // Remove x-restbase and x-rb headers if the request is not local
+    if (!/^(?:::ffff:)?(?:10|127)\./.test(remoteAddr) && newReq.headers) {
```

Comment:
Minor: It might be nice test for local / remote once. Perhaps move this to line 204?